features/step_definitions/storage_class.rb

```diff
@@ -189,7 +189,8 @@
      sc_hash.dig("metadata", "annotations", "storageclass.beta.kubernetes.io/is-default-class") == "true"
     ensure_destructive_tagged
   end
-
+  # Update sc to Immediate for smooth testing
+  sc_hash["volumeBindingMode"] = "Immediate"
```

Comment:
We need to put it before `table.raw.each...`, e.g, after `sc_hash["metadata"]["name"] = "#{target_sc}"`

Reply:
i.e. you want to set a default but be able to override in table. +1